_topic_maps/_topic_map.yml

```diff
@@ -501,8 +501,8 @@ Topics:
     File: installing-customizing
   - Name: Configuring your firewall
     File: configuring-firewall
-  - Name: Enabling Linux control group version 2 (cgroup v2)
-    File: enabling-cgroup-v2
+  - Name: Enabling Linux control group version 1 (cgroup v1)
+    File: enabling-cgroup-v1
```

Comment:
Since the file name is changing, that means anyone who was aware of this file existing in 4.13 trying to find the same path in 4.14 would get an error. We can set up a redirect though so that someone looking in 4.14 docs for the old path will just end up on the new file. I'll ping you on Slack for details.

Reply:
done